.github/pull_request_template.md

```diff
@@ -12,8 +12,7 @@ please make sure you've corresponded with the repo's maintainer to
 ensure it's work that is currently prioritized and budgeted.
 
 You can do this by opening an issue explaining your proposed changes and
-cost, or by floating the idea in the #website channel on 
-Keybase (https://keybase.io/team/bisq).
+cost, or by floating the idea in the Growth channel on Matrix (https://bisq.chat).
```

Comment:
```suggestion
cost, or by floating the idea in the Growth channel on Matrix (https://matrix.to/#/#bisq.growth:bitcoin.kyoto).
```

Reply:
Yes, makes sense. Didn't know that this works.